src/compiler/python_generator.cc

```diff
@@ -101,6 +101,12 @@ PrivateGenerator::PrivateGenerator(const GeneratorConfiguration& config,
 void PrivateGenerator::PrintAllComments(StringVector comments,
                                         grpc_generator::Printer* out) {
   if (comments.empty()) {
+    // Print a blank comment anyway. This is important in Python, because
+    // class Service:
+    //
+    // is invalid unless supplanted with `pass` or some body.
+    // The blank comment satisfies having a body requirement.
+    out->Print("\"\"\"\"\"\"");
```

Comment:
An empty doc string is worse than no doc string at all. How about just the `pass` statement? Maybe with an octothorpe-delimited comment saying... something?